packages/ses/README.md

```diff
@@ -32,6 +32,12 @@ SES starts where the Caja project left off
 https://github.com/google/caja/wiki/SES, and goes on to introduce compartments
 and modernize the permitted JavaScript features.
 
+Please [join the conversation on Matrix](https://matrix.to/#/#endojs:matrix.org).
+
+[![Primer on Hardened JavaScript](https://img.youtube.com/vi/RoodZSIL-DE/0.jpg)](https://www.youtube.com/watch?v=RoodZSIL-DE)
+
+[![Using `ses` to reduce supply chain risk](https://img.youtube.com/vi/88fetnmJvps/0.jpg)](https://www.youtube.com/watch?v=88fetnmJvps)
```

Comment:
I know I suggested this one earlier, but I think https://www.youtube.com/watch?v=u-XETUbxNUU&list=PLKr-mvz8uvUgybLg53lgXSeLOp4BiwvB2 may be better. What do you think?